php_phongo.c

```diff
@@ -957,36 +957,36 @@ bool phongo_execute_command(mongoc_client_t* client, php_phongo_command_type_t t
 		goto cleanup;
 	}
 
-	/* According to mongoc_cursor_new_from_command_reply(), the reply bson_t
-	 * is ultimately destroyed on both success and failure. */
+	/* According to mongoc_cursor_new_from_command_reply_with_opts(), the reply
+	 * bson_t is ultimately destroyed on both success and failure. */
 	if (bson_iter_init_find(&iter, &reply, "cursor") && BSON_ITER_HOLDS_DOCUMENT(&iter)) {
 		bson_t       initial_reply = BSON_INITIALIZER;
 		bson_error_t error         = { 0 };
 
 		bson_copy_to(&reply, &initial_reply);
 
 		if (command->max_await_time_ms) {
-			bson_append_bool(&initial_reply, "awaitData", -1, 1);
-			bson_append_int64(&initial_reply, "maxAwaitTimeMS", -1, command->max_await_time_ms);
-			bson_append_bool(&initial_reply, "tailable", -1, 1);
+			bson_append_bool(&opts, "awaitData", -1, 1);
```

Comment:
At this point `opts` is the original document we used to build options for `mongoc_client_command_with_opts()` (and related functions). I think we should start with a new `opts` document for `mongoc_cursor_new_from_command_reply_with_opts()` (e.g. `cursor_opts`), as not all options are shared.

This re-use may be why server-pinning was working (related to me asking to set `serverId` on `opts` below), but that's not terribly clear.